src/components/Navbar.vue

```diff
@@ -0,0 +1,125 @@
+import { RouterLink, RouterView } from "vue-router";
+
+export default {
+<template>
+  <header>
+    <div class="left-side">
+      <img
```

Comment:
Let's use the v-img like in Toolhub for the logo and the image should also be a link to the home page. https://github.com/wikimedia/toolhub/blob/main/vue/src/App.vue#L24-L30

Reply:
Good catch - I've enclosed both the icon and "Toolhunt" in a router link to the home page, which mimics what happens on Toolhub.